bin/main/org/fundacionjala/at15/katas/README.md

```diff
@@ -0,0 +1,43 @@
+# Katas folder
```

Comment:
Hi @alvarogerardo007 , please remove bin/ folder and include it as part of `.gitignore` file. You can remove bin/ with `git rm -r bin`